administration/src/bp-modules/self-service/hooks/useCardGeneratorSelfService.ts

```diff
@@ -27,26 +25,25 @@ type UseCardGeneratorSelfServiceReturn = {
   isLoading: boolean
   deepLink: string
   code: CreateCardsResult | null
-  selfServiceCards: SelfServiceCard[]
-  setSelfServiceCards: (cards: SelfServiceCard[]) => void
+  selfServiceCard: CardBlueprint
+  setSelfServiceCard: (card: CardBlueprint) => void
   generateCards: () => Promise<void>
   downloadPdf: (code: CreateCardsResult, fileName: string) => Promise<void>
 }
-const initializeCardBlueprintForProject = (projectConfig: ProjectConfig): SelfServiceCard => {
-  const headers = getHeaders(projectConfig)
-  const selfServiceCard = new SelfServiceCard('', projectConfig.card)
-  headers.forEach(header => {
-    selfServiceCard.setValue(header, '')
-  })
-  return selfServiceCard
-}
```

Comment:
Why was this necessary? It is just setting empty values anyway?

Reply:
because of the `isValid()` and `isFullnameValid()` fkts. Currently they will be executed directly because we have no untouched state